### ISSUE TITLE
Updated atrcopy with pip install atrcopy, now it just crashes. 

### ISSUE THREAD

**Issue by ChoccyHobNob** [robmcmullen/atrcopy#11]
I updated today and now I just get this

```
atrcopy /Users/steve/Downloads/atrfilestest/5-dos.atr 
Traceback (most recent call last):
  File "/usr/local/bin/atrcopy", line 4, in <module>
    import atrcopy
  File "/usr/local/lib/python3.6/site-packages/atrcopy/__init__.py", line 355
    except InvalidDiskImage, e:
                           ^
SyntaxError: invalid syntax
```

pip removed atrcopy and pip installed it again, it's still the same.

**Comment by robmcmullen** [comment]
I pushed atrcopy 7.0: https://pypi.org/project/atrcopy/7.0/

but pip install isn't refreshing yet. Maybe it needs some time to propagate. Give that a try when you get a chance.

**Comment by robmcmullen** [comment]
Actually, try version 7.1. It should fix the spartados issue (#6) you were having

**Comment by ChoccyHobNob** [comment]
That fixed it, thanks.

7.1 will list SpartaDOS disks but wont extract them, it prints the disk header and quits gracefully so a massive improvement from the Traceback. 
dcm files just report a single line "0 [basic].dcm: BSAVE data $41fa-$41fb ($0001 @ $0004)"
xfd files behave just like atrs
Is that the expected behaviour now? 

Big improvement, awesome work!
